docs/website/01_framework/controlRoutines.md

```diff
@@ -167,3 +173,21 @@ struct StandardArcLength {
     std::optional<double> psi;
   };
 ```
+
+## Automatic step-sizing for path-following techniques
+
+Instead of using a constant step size, the step size can be automatically adapted for efficient computations.
+The automatic step-sizing is implemented according to Ramm[@wunderlich_strategies_1981] .
+The step size can be scaled as shown below:
+
+$$
+\hat{s}_{k+1} = \left(\frac{\hat{i}}{i_k}\right)^{1/2}\hat{s}_k
+$$
+
+Here, $\hat{s}_{k+1}$ and $\hat{s}_{k}$ are the step sizes at $k+1$-th and $k$-th iteration.
+Here, $\hat{i}$ is the desired number of iterations and $i_k$ is the number of iterations used in the previous step.
+
+For implementation details, refer to `ikarus/controlroutines/adaptivestepsizing.hh`.
+
+By default, `NoAdaptiveStepSizing` is used with a path-following technique.
+The `DefaultAdaptiveStepSizing` can be used as a step sizing method, if required.
```

Comment:
You should rename `DefaultAdaptiveStepSizing` to something else.
Its not descriptive, nor is it the default.